src/components/material/material-buttons/online/MaterialButtonsOnline.tsx

```diff
@@ -40,7 +41,9 @@ const MaterialButtonsOnline: FC<MaterialButtonsOnlineProps> = ({
       trackedData: workId
     });
   };
-  const { type: readerPlayerType } = useReaderPlayer(manifestations);
+  const { type: readerPlayerType } = useReaderPlayer(
+    getFirstManifestation(manifestations)
+  );
```

Comment:
Can this be reduced to `const readerPlayerType = getReaderPlayerType(etFirstManifestation(manifestations)`.

As far as I can tell there is little value in using the hook here.